### DISCUSSION TITLE
CommandBarFlyout has border overlapping background

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/microsoft-ui-xaml#7417] [General]
All other menus and flyouts have a semi-transparent border outside of the actual menu, but for CommandBarFlyout it's applied on top of the background. You can see it here (left) compared to a MenuFlyout (right):
![image](https://user-images.githubusercontent.com/99533763/153706344-fe33bc1d-6f8b-4236-968d-bba7c9e81383.png)

The divider between the primary and secondary panels is just the border of the top panel, so it's not easy to fix. Simply moving the border outside of the background will leave a gap there. The divider would ideally be separated from the actual border, which would allow customizing it independently (color, thickness etc).

**Comment by StephenLPeters** [comment]
@chigy FYI
